src/wallet/wallet.h

```diff
@@ -268,7 +271,7 @@ class CMerkleTx
 //Get the marginal bytes of spending the specified output
 int CalculateMaximumSignedInputSize(const CTxOut& txout, const CWallet* pwallet);
 
-/** 
```

Comment:
I can remove these whitespace change if that is controversial.

Reply:
no - please don't invalidate reviewer ACKs unnecessarily